src/cephadm/cephadm

```diff
@@ -2569,6 +2569,29 @@ def command_bootstrap():
     cp.write(cpf)
     config = cpf.getvalue()
 
+    if args.registry_json:
+        logger.info(f"Pulling custom registry login info from {args.registry_json}.")
```

Comment:
http://qa-proxy.ceph.com/teuthology/swagner-2020-07-21_10:29:02-rados:cephadm-wip-swagner-testing-2020-07-21-1046-distro-basic-smithi/5247136/teuthology.log

```
2020-07-21T10:57:02.876 INFO:tasks.workunit.client.0.smithi072.stderr:    logger.info(f"Pulling custom registry login info from {args.registry_json}.")
2020-07-21T10:57:02.876 INFO:tasks.workunit.client.0.smithi072.stderr:                                                                               ^
2020-07-21T10:57:02.877 INFO:tasks.workunit.client.0.smithi072.stderr:SyntaxError: invalid syntax
```

http://qa-proxy.ceph.com/teuthology/swagner-2020-07-21_10:29:02-rados:cephadm-wip-swagner-testing-2020-07-21-1046-distro-basic-smithi/5247136/teuthology.log

can you please add a test (either pytest or teuthology or someting) that exercises at least one code path below?